### PR TITLE
Add Netflix story

### PR DESCRIPTION
**Pull request by fpapon**

### CODE REVIEW

File: src/main/webapp/stories.html [hunk 1]
```diff
@@ -175,15 +175,30 @@ <h4 class="pb-3 mb-4">Fuse</h4>
...
               <p>Digital Banking Platform from Novabase.</p>
               <a href="http://www.novabase.com/financial/wizzio/" target="_blank">http://www.novabase.com/financial/wizzio/</a>
             </div>
         </div>
-        
+
+        <div class="row mb-5 mt-5">
+          <div class="col">
+            <h4 class="pb-3 mb-4">Netflix</h4>
+            <p><i>The Netflix media pipeline processes thousands of new shows and movies every day so that you can watch them on any device anywhere.
```
Comment (jbonofre) [inline src/main/webapp/stories.html:190]: Maybe worth to mention the talk at ApacheCon.

Comment (fpapon) [reply src/main/webapp/stories.html:190]: @jbonofre right, may be adding the link to the talk schedule?

Comment (jbonofre) [reply src/main/webapp/stories.html:190]: Yes, as soon as the schedule will be online.

Comment (svogt) [reply src/main/webapp/stories.html:190]: https://www.apachecon.com/acna19/s/#/scheduledEvent/1209

Comment (jbonofre) [reply src/main/webapp/stories.html:190]: @fpapon mind to update the PR with link to ApacheCon event ?

Comment (fpapon) [reply src/main/webapp/stories.html:190]: @jbonofre always done on latest force push :)

**Review by jbonofre — APPROVED** [review]

**Pull request merged**